Containerfile

```
@@ -1,5 +1,9 @@
-FROM docker.io/debian
-RUN apt update && apt -y upgrade
-RUN apt install -y make texlive-latex-recommended
-RUN apt install -y texlive-latex-extra
+FROM docker.io/fedora:38
+RUN dnf -y upgrade
+RUN dnf -y install texlive-scheme-basic
+RUN dnf -y install make
+RUN dnf -y install texlive-ulem texlive-multirow texlive-fancybox texlive-mdwtools
+RUN dnf -y install texlive-ccicons
+RUN dnf -y install ghostscript
+RUN dnf -y install librsvg2-tools
```

Container is missing `inkscape` binary required for `front-cover.pdf` makefile target.

```suggestion
RUN dnf -y install librsvg2-tools inkscape
```

---

It is intentional. Inside the container, `rsvg-tools` does the front cover generation (which looks pretty much like the current cover). If we install `inkscape` inside the container, it will be wayyy too bloated. So the Makefile recipe for `front-cover.pdf` does a little branching.